app/javascript/dashboard/assets/scss/widgets/_modal.scss

```diff
@@ -89,6 +86,11 @@
     button {
       font-size: $font-size-small;
     }
+
+    &.justify-content-end {
```

Comment:
I think the class name is misleading, Can we either rename class by it's nature or split into two classes?